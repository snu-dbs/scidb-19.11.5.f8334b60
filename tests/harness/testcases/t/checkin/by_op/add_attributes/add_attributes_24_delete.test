--setup
--start-query-logging
create array f<v:int64>[i=0:4:0:1]
store(build(f, i), f)

--test
add_attributes(f, <w:int64>)
scan(f)
store(project(apply(f, wnew, 2*v+1), v, wnew), f)
scan(f)
delete(f, v=2)
scan(f)

--cleanup
remove(f)
--stop-query-logging
