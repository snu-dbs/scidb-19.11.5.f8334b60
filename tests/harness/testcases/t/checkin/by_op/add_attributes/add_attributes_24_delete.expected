SCIDB QUERY : <create array f<v:int64>[i=0:4:0:1]>
Query was executed successfully

SCIDB QUERY : <store(build(f, i), f)>
Query was executed successfully

SCIDB QUERY : <add_attributes(f, <w:int64>)>
Query was executed successfully

SCIDB QUERY : <scan(f)>
{i} v,w
{0} 0,null
{1} 1,null
{2} 2,null
{3} 3,null
{4} 4,null

SCIDB QUERY : <store(project(apply(f, wnew, 2*v+1), v, wnew), f)>
Query was executed successfully

SCIDB QUERY : <scan(f)>
{i} v,w
{0} 0,1
{1} 1,3
{2} 2,5
{3} 3,7
{4} 4,9

SCIDB QUERY : <delete(f, v=2)>
Query was executed successfully

SCIDB QUERY : <scan(f)>
{i} v,w
{0} 0,1
{1} 1,3
{3} 3,7
{4} 4,9

SCIDB QUERY : <remove(f)>
Query was executed successfully

